#include <iostream>
#include <stdexcept>

using namespace std;

int main() {
	throw runtime_error("Error!");
}
